signer/contentsignaturepki/contentsignature_test.go

```diff
@@ -76,15 +76,21 @@ func TestSign(t *testing.T) {
 		if err != nil {
 			t.Fatalf("testcase %d failed to get X5U %q: %v", i, s.X5U, err)
 		}
-		key := certs[0].PublicKey.(*ecdsa.PublicKey)
+		leaf := certs[0]
+		key := leaf.PublicKey.(*ecdsa.PublicKey)
 		if !sig.(*verifier.ContentSignature).VerifyData([]byte(input), key) {
 			t.Fatalf("testcase %d failed to verify signature", i)
 		}
+
+		if leaf.Subject.CommonName != testcase.expectedCommonName {
+			t.Errorf("testcase %d expected common name %#v, got %#v", i, testcase.expectedCommonName, leaf.Subject.CommonName)
+		}
 	}
 }
 
 var PASSINGTESTCASES = []struct {
-	cfg signer.Configuration
+	cfg                signer.Configuration
+	expectedCommonName string
```

Comment:
If I'm reading this correctly, we modified the existing test case for the new code. I'd prefer that we add a new test case to show that the "old config format" (without the `SubdomainOverride` field) still works.

Reply:
Good call. Done